examples/benchmarks/bert/glue.py

```diff
@@ -13,46 +13,105 @@
 from typing import Any, Dict, List, Optional, Sequence, Set, Tuple
 from urllib.parse import urlparse
 
+sys.path.append(os.path.dirname(os.path.realpath(__file__)))
```

Comment:
can we put a comment where we do this (like what I did for tests with scripts in LLM foundry?)

Reply:
will do

Reply:
added comments, and also fixed a few of them